homeassistant/components/huum/config_flow.py

```diff
@@ -54,7 +54,9 @@ async def async_step_user(
                 await self.async_set_unique_id(
                     user_input[CONF_USERNAME], raise_on_progress=False
                 )
```

Comment:
```suggestion
```
You can remove this one as you're now aborting based on entry data. In the future we could use the unique id here

Reply:
Alright, I'll remove this 🙂

Reply:
Fixed